test-data/unit/stubgen.test

```diff
@@ -4249,6 +4247,51 @@ o = int | None
 def f1(a: int | tuple[int, int | None] | None) -> int: ...
 def f2(a: int | x.Union[int, int] | float | None) -> int: ...
 
+[case testTypingBuiltinReplacements]
+import typing
+import typing as t
+from typing import Tuple
+import typing_extensions
+import typing_extensions as te
+from typing_extensions import List, Type
+
+# builtins are not builtins
+tuple = int
```

Comment:
Can you also define `_tuple` here to make sure we handle that correctly? If I recall correctly we just keep adding more underscores in the generated stub.
```suggestion
tuple = int
_tuple = range
```

Reply:
> If I recall correctly we just keep adding more underscores in the generated stub.

Turns out we weren't. It didn't cause a lot of trouble because as long as you didn't use the `--include-private` flag, prefixing the name with a single `_` was enough.
Fixed in cdbde86.